src/OpenTelemetry.Exporter.Zipkin/ZipkinExporter.cs

```diff
@@ -119,7 +120,8 @@ internal void SetLocalEndpointFromResource(Resource resource)
 
             if (string.IsNullOrEmpty(serviceName))
             {
-                serviceName = this.options.ServiceName;
+                serviceName = (string)this.ParentProvider.GetDefaultResource().Attributes.Where(
```

Comment:
Please add to Zipkin changelog:
"Removes ability to configure ServiceName for Zipkin. ServiceName must come via Resource. If service name is not found in Resource, Zipkin uses GetDefaultResource API from the SDK."